cola-tests/src/main/java/com/github/bmsantos/core/cola/utils/ColaUtils.java

```diff
@@ -66,11 +66,11 @@ public static String binaryToOS(final String binaryFormat) {
     }
 
     public static String binaryToOsClass(final String binaryFormat) {
-        String result = binaryToOS(binaryFormat);
-        if (!isClassFile(result)) {
-            result += CLASS_EXT;
+        StringBuilder result = new StringBuilder(binaryToOS(binaryFormat));
+        if (!isClassFile(result.toString())) {
```

Comment:
Please fix!

This is not required in this case. We might as well remove or forbid the "+" operator from being used in Java.